doc/input-output-reference/src/overview/group-air-distribution-equipment.tex

```diff
@@ -63,6 +63,17 @@ \subsubsection{Inputs}\label{inputs-1-001}
 
 This is the upper limit on the temperature in degrees C of the air leaving the terminal unit -- reheat coil (and being delivered to the zone). If the user leaves this field blank, no maximum reheat air temperature is enforced.
 
+\subsubsection{Outputs}
+
+\begin{itemize}
+\item
+  HVAC,Average,Zone Air Terminal Outdoor Air Volume Flow Rate {[}m3/s{]}
+\end{itemize}
+
+\paragraph{Zone Air Terminal Outdoor Air Volume Flow Rate {[}m3/s{]}}\label{zone-air-terminal-outdoor-air-volume-flow-rate-m3s}
+
+This output is the amount of outdoor air entering the zone. This is the average value over the frequency being reported. The amount of outdoor air is defined as the terminal unit air volume flow rate multiplied by the fraction of outdoor air entering the air loop's outside air system.
+
```

Comment:
This is description of the new report variable added for AirTerminal:SingleDuct:ConstantVolume:Reheat air terminal unit.